toolset/setup/linux/languages/hhvm.sh

```diff
@@ -12,10 +12,7 @@ fi
 
 fw_get http://dl.hhvm.com/conf/hhvm.gpg.key | sudo apt-key add -
 
-# hhvm 3.24 causes most of our hhvm-using test implementations to segmentation
-# fault when connecting to the mysql database (the crash happens right as they
-# try to specify utf8 as the charset).  hhvm 3.21 doesn't have this problem.
-echo deb [arch=amd64] http://dl.hhvm.com/ubuntu `lsb_release -sc`-lts-3.21 main | sudo tee /etc/apt/sources.list.d/hhvm.list
```

Comment:
You're reverting a fix we needed to make here. We will have to test locally but probably can't accept this change.

Reply:
Apparently that bug in hhvm 3.24 that we were working around is fixed in 3.24.2: https://hhvm.com/blog/2018/02/07/hhvm-3.24.2.html

Reply:
Oh good!

Reply:
Yes, I wrote:
HHVM 3.21 -> 3.24 (Fixed crash with utf-8 and mysql)
I will add the link too.

I tested the benchmarks locally without problems, before submitting the commit.